src/_gettsim/transfers/rente.py

```diff
@@ -695,6 +783,59 @@ def ges_rente_vorauss_frauen(  # noqa: PLR0913
         and ges_rente_wartezeit_15 >= 15
         and y_pflichtbeitr_ab_40 >= 10
         and alter >= altersgrenze_vorzeitig
+        and geburtsjahr < 1952
+    )
+
+    return out
+
+
+def _ges_rente_vorauss_arbeitslos(  # noqa: PLR0913
+    arbeitsl_1y_past_585: bool,
+    ges_rente_wartezeit_15: float,
+    pflichtbeitr_8_in_10: bool,
+    birthdate_decimal: float,
+    ges_rente_params: dict,
+    alter: int,
+    #  @TeBackh: need to fix to monthly precision! change to age_of_retirement
+) -> bool:
+    """Function determining the eligibility for Altersrente für Arbeitslose (pension
+    for unemployed. Wartezeit 15 years, 8 contributionyears past 10 years, being
+    at least 1 year unemployed after age 58 and 6 months.
+
+    Parameters
+    ----------
+    arbeitsl_1y_past_585
+        See basic input variable
+        :ref:`arbeitsl_1y_past_585 <arbeitsl_1y_past_585>`.
+    ges_rente_wartezeit_15
+        See :func:`ges_rente_wartezeit_15`
+    pflichtbeitr_8_in_10
+        See basic input variable :ref:`pflichtbeitr_8_in_10 <pflichtbeitr_8_in_10>`.
+    birthdate_decimal
+        See :func:`birthdate_decimal`
+    ges_rente_params
+        See params documentation :ref:`ges_rente_params <ges_rente_params>`.
+
+    Returns
+    -------
+    Eligibility as bool.
+
+    """
+    altersgrenze_vorzeitig = piecewise_polynomial(
+        x=birthdate_decimal,
+        thresholds=ges_rente_params["altergrenze_arbeitslose_vorzeitig"]["thresholds"],
+        rates=ges_rente_params["altergrenze_arbeitslose_vorzeitig"]["rates"],
+        intercepts_at_lower_thresholds=ges_rente_params[
+            "altergrenze_arbeitslose_vorzeitig"
+        ]["intercepts_at_lower_thresholds"],
+    )
+
+    out = (
+        arbeitsl_1y_past_585
+        and ges_rente_wartezeit_15 >= 15
```

Comment:
Same as above, no hard-coded numbers, please.

Reply:
replaced most hardcoded numbers with parameters (except 1, 12 and 0)